includes/admin/class-wc-payments-admin-sections-overwrite.php

```diff
@@ -8,15 +8,24 @@
 defined( 'ABSPATH' ) || exit;
 
 /**
- * WC_Payments_Admin Class.
+ * WC_Payments_Admin_Sections_Overwrite Class.
  */
 class WC_Payments_Admin_Sections_Overwrite {
+	/**
+	 * Used for restoring the original value of `$current_section` global variable.
+	 *
+	 * @var string
+	 */
+	private $previous_current_section;
+
 	/**
 	 * WC_Payments_Admin_Sections_Overwrite constructor.
 	 */
 	public function __construct() {
 		add_action( 'woocommerce_settings_page_init', [ $this, 'page_init' ] );
 		add_filter( 'woocommerce_get_sections_checkout', [ $this, 'add_checkout_sections' ] );
+		add_action( 'woocommerce_sections_checkout', [ $this, 'overwrite_current_section_global' ], 5 );
```

Comment:
:( sad that we needed to re-add them

Reply:
Thanks to your previous work, this was a quick change!